<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombatForm</class>
 <widget class="QWidget" name="CombatForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>490</y>
     <width>1281</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hand_container"/>
  </widget>
  <widget class="QLabel" name="player_life">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Player Life</string>
   </property>
  </widget>
  <widget class="QLabel" name="enemy_life">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enemy Life</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
